h2/src/main/org/h2/expression/SequenceValue.java

```diff
@@ -108,4 +108,4 @@ public int getCost() {
         return 1;
     }
 
-}
+}
```

Comment:
Now you lost an empty newline at the end of this file.

Not very critical, but this file is not related with your changes and shouldn't be touched at all.

Reply:
I can't imagine why the merge problems came up. I never intended to change anything at this file. I hope it is now in the original state.